### ISSUE TITLE
Current status?

### ISSUE THREAD

**Issue by YouCanNotBeSerious** [lincomatic/LeafCAN#2]
Hi - 

I understand this is a dig, given the dates of last commits :-)

I am looking for a way to read SoC on a 2018 (ZE1) Leaf, with my idea being a very simple headless ESP32 hardware implementation that sends current SoC to a webserver every x minutes.  Any hints/tips on whether I should try your project?  I am perfeclty willing to stand on the shoulders of giants and get my hands dirty testing, but you might for example say there's a fundamental incompatibility with second gen Leafs.

Thanks!

**Comment by PaulKennett** [comment]
Yes. Totally doable. You'll need to tweak the code for ZE1 CAN messages. See https://github.com/dalathegreat/leaf_can_bus_messages
